apps/vercel/frontend/src/components/config-screen/ContentTypePreviewPathSection/ContentTypePreviewPathSelectionList/ContentTypePreviewPathSelectionList.tsx

```diff
@@ -0,0 +1,91 @@
+import { ContentType } from 'contentful-management';
+import { PlusIcon } from '@contentful/f36-icons';
+import { Button } from '@contentful/f36-components';
+import { useState, Dispatch, SetStateAction } from 'react';
+
+import { ContentTypePreviewPathSelectionRow } from '../ContentTypePreviewPathSelectionRow/ContentTypePreviewPathSelectionRow';
+import { ContentTypePreviewPathSelection } from '../../../../types';
+import { actions } from '../../../parameterReducer';
+
+interface Props {
+  contentTypes: ContentType[];
+  dispatchParameters: Dispatch<SetStateAction<any>>;
+  contentTypePreviewPathSelections: ContentTypePreviewPathSelection[];
+}
+
+export const ContentTypePreviewPathSelectionList: React.FC<Props> = ({
+  contentTypes,
+  dispatchParameters,
+  contentTypePreviewPathSelections = [],
+}) => {
+  const [addRow, setAddRow] = useState<number[]>([]);
+
+  const handleUpdateParameters = (parameters: ContentTypePreviewPathSelection) => {
+    if (parameters.contentType && parameters.previewPath) {
+      if (addRow.length) setAddRow([]);
+      dispatchParameters({
+        type: actions.ADD_CONTENT_TYPE_PREVIEW_PATH_SELECTION,
+        payload: parameters,
+      });
+    }
+  };
+
+  const handleRemoveRow = (parameters: ContentTypePreviewPathSelection) => {
+    if (addRow.length) setAddRow([]);
+
+    dispatchParameters({
+      type: actions.REMOVE_CONTENT_TYPE_PREVIEW_PATH_SELECTION,
+      payload: parameters,
+    });
+  };
+
+  // TO DO: Adjust AddRow logic to boolean if we want to enforce only one row appearing at once
```

Comment:
notes in the PR description, will adjust this logic once we experiment with the workflow of when the Add ContentType button appears i.e if users can endlessly add this, if it is disabled until the user has filled out the current row etc. For now, a user can only add one row at a time, and it is disabled if the current row is empty.